administration/src/bp-modules/applications/components/VerificationListItem.tsx

```diff
@@ -59,6 +90,17 @@ const VerificationListItem = ({ verification }: VerificationListItemProps): Reac
           </tr>
         </tbody>
       </table>
+      {authData !== null && status === VerificationStatus.Pending && (
```

Comment:
I wanted to hide the button on the applicant view (/antrag-einsehen) but not sure what was the optimal way

Reply:
sounds good. thanks for the idea, applied